string state_name
float32 population